src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/SqlBulkCopyTest/CopyAllFromReader.cs

```diff
@@ -60,7 +60,7 @@ public static void Test(string srcConstr, string dstConstr, string dstTable)
                             DataTestUtility.AssertEqualsWithDescription((long)0, (long)stats["CursorOpens"], "Non-zero CursorOpens value: " + (long)stats["CursorOpens"]);
                             DataTestUtility.AssertEqualsWithDescription((long)0, (long)stats["IduRows"], "Non-zero IduRows value: " + (long)stats["IduRows"]);
 
-                            DataTestUtility.AssertEqualsWithDescription((long)4, stats["BuffersReceived"], "Unexpected BuffersReceived value.");
+                            DataTestUtility.AssertEqualsWithDescription((long)3, stats["BuffersReceived"], "Unexpected BuffersReceived value.");
```

Comment:
Why ?

Reply:
Can you explain why this test case changed

Reply:
This happens because SQL Server sends Acknowledgement of ATTN Signal (with DONE_ATTN Token) which upon fixing this issue is not received.

Reply:
I see. Thanks